transaction/core/src/validation/validate.rs

```diff
@@ -510,9 +510,11 @@ mod tests {
     use mc_crypto_keys::{CompressedRistrettoPublic, ReprBytes};
     use mc_ledger_db::{Ledger, LedgerDB};
     use mc_transaction_core_test_utils::{
-        create_ledger, create_transaction, create_transaction_with_amount, initialize_ledger,
-        AccountKey, INITIALIZE_LEDGER_AMOUNT,
+        create_ledger, create_transaction, create_transaction_with_amount_and_comparer,
+        initialize_ledger, outputs_comparer_inverse, outputs_comparer_normal, AccountKey,
```

Comment:
2 name change suggestions: 
1. outputs_comparer_inverse --> inverse_outputs_comparer 
2. create_transaction_with_amount could probably stay the same given it has an optional parameter

Reply:
changed to traits, naming changed